Formula/freedink-data.rb

```diff
@@ -0,0 +1,18 @@
+class FreedinkData < Formula
+  desc "Data for FreeDink."
```

Comment:
You can make this a resource of the `freedink` formula instead of a formula in itself.